static/data/team.fr.json

```diff
@@ -0,0 +1,392 @@
+{
+  "team": {
+    "poles": [
+      {
+        "name": "Direction Générale",
+        "members": [
+          {
+            "name": "Emmanuel Aldeguer",
+            "photo": "https://pollen-by-okp4.com/wp-content/uploads/2023/01/Emmanuel_Aldeguer.png.webp",
```

Comment:
Relying on external URLs to obtain images is not a good idea since the location of the resources may be altered. So I'd recommend to use static images instead.

To help in downloading all images of this file, you can use this one-liner (needs [jq](https://stedolan.github.io/jq/) and [wget](https://www.gnu.org/software/wget/)):

```sh
jq -r '.team.poles[].members[].photo' < team.json | xargs wget
```

Reply:
I made the change : 

- I downloaded the images and placed them into `static/img/team` folder.
- I renamed them by the name of the team member
- I removed the `photo` property in the json file
- I displayed the image by this way : 
```tsx 
   <img
      src={`/img/team/${member.name}.webp`}
   />
```